homeassistant/components/androidtv/media_player.py

```diff
@@ -278,13 +279,13 @@ def __init__(self, aftv, name, apps, turn_on_command, turn_off_command):
 
         # ADB exceptions to catch
         if not self.aftv.adb_server_ip:
-            # Using "python-adb" (Python ADB implementation)
-            from adb.adb_protocol import (
+            # Using "adb_shell" (Python ADB implementation)
+            from adb_shell.exceptions import (
```

Comment:
Can we move the imports to the top of the module?

Reply:
Done.